Automattic/simplenote-macos
# Bugfix: Editor Scroller Insets

### Fix
In this PR we're adjusting the Editor's Scroller Insets, so that they match the Notes List insets (and don't appear to be clipped off).

Ref. #718
cc @eshurakov (SPAMMING!!!) 😂 
Thank you Evgeny!!

### Test
1. Open your macOS `System Preferences` > `General`
2. Enable **Show Scroll Bars >> Always**
3. Launch Simplenote

- [ ] Verify the Notes List Scrollbar matches the location of the Editor's Scrollbar
- [ ] Verify the Editor's Scrollbar starts below the Toolbar area (and does not appear to be clipped off)

### Release
`RELEASE-NOTES.txt` was updated in 170ae2b with:

> Fixed a bug that caused the Editor's Scrollbar to get clipped #777

### Screenshots
Old Mojave|New Mojave
-|-
<img width="300" alt="Old Mojave" src="https://user-images.githubusercontent.com/1195260/103308525-67104e80-49f1-11eb-82eb-91e40059ff67.png">|<img width="300" alt="New Mojave" src="https://user-images.githubusercontent.com/1195260/103308530-6aa3d580-49f1-11eb-8156-e4772c711042.png">

Old Big Sur|New Big Sur
-|-
<img width="300" alt="Old Big Sur" src="https://user-images.githubusercontent.com/1195260/103308552-77282e00-49f1-11eb-9a9b-bf0b31f4509e.png">|<img width="300" alt="New Big Sur" src="https://user-images.githubusercontent.com/1195260/103308561-7abbb500-49f1-11eb-943b-c2d1e8552aaf.png">
